rotkehlchen/api/rest.py

```diff
@@ -296,6 +296,17 @@ def wrap_in_fail_result(message: str, status_code: HTTPStatus | None = None) ->
     return result
 
 
+def maybe_get_temp_file(filepath: FileStorage | Path) -> Path:
```

Comment:
is only used in this file

```suggestion
def _maybe_get_temp_file(filepath: FileStorage | Path) -> Path:
```